src/self-help/README.md

```diff
@@ -20,7 +20,7 @@ az extension add --name self-help
 
   ```
   # Gets list of solution metadata for an azure resource.
-  az self-help discovery-solution list --scope {scope}
+  az self-help discovery-solution list --filter "ProblemClassificationId eq '00000000-0000-0000-0000-000000000000'" --scope 'subscriptions/0d0fcd2e-c4fd-4349-8497-200edb3923c6/resourceGroups/myresourceGroup/providers/Microsoft.KeyVault/vaults/test-keyvault-non-read'
```

Comment:
Please replace subscription id by `00000000-0000-0000-0000-000000000000` in all examples.

Reply:
> Is `0d0fcd2e-c4fd-4349-8497-200edb3923c6` a valid subscription? If it's a valid subid, please replace it by `00000000-0000-0000-0000-000000000000` in all examples.

@kairu-ms It is an auto generated fake guid, but I have updated subscription in all examples as suggested. Could you please help us review this again? and resolve comments if they are fixed?